flow-typed/npm/@alrra/travis-scripts_vx.x.x.js

```diff
@@ -1,5 +1,5 @@
-// flow-typed signature: 2d24c01706b62f06b761f21d82b05461
```

Comment:
you can ignore all the `flow-typed/` changes - just keeping Flow types up to date